### ISSUE TITLE
Settings> Contact us - Accessibility issues

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#433]
Related to #239 

Settings > Contact us
The focus should be "Go back" button > Heading "Contact us"> Rest of the content.

Important: When the focus goes to the bottom, it should identify "Toolbar" > "Back button" > "Forward button". If the button is disabled it should say "dimmed".

**Comment by Jiandong-Guo** [comment]
@giftydeep ,Get latest and try test again

**Comment by Rajat-Sen** [comment]
It's working, 

however, please make sure we are using the correct labels.

< - Back dimmed button (If greyed out)
> - Forward dimmed button (if greyed out)

If none of these buttons are greyed out, it should skip the word dimmed and only say back button and forward button. Presently, it's wrongfully labelled as RightArrowButton.

Thanks!

**Comment by giftydeep** [comment]
@Rajat-Sen We are not setting any labels from ourself. Because Voiceover in iPhone is very smart so it pick up thing by itself and we have very less control in terms of Accessibility. Back and right arrow is also picked up by voiceover as these icons are called back and right arrow. However i will try to find the icon with name Forward so it can be picked up as forward. if not we can't really update these labels.

**Comment by Rajat-Sen** [comment]
@giftydeep, have you found correct labelled button?

**Comment by giftydeep** [comment]
@Rajat-Sen the icons I found doesn't load properly. As this is not a priority ticket so either we can put this on Hold or close this ticket for now and I will look into it after the new features.

**Comment by Rajat-Sen** [comment]
Okay, putting it on hold for now!
